### ISSUE TITLE
WCAG Accessibility

### ISSUE THREAD

**Issue by kyleadministrate** [sstur/react-rte#383]
Are there plans to make react-rte compliant with WCAG accessibility? It currently fails the axe checks for a variety of reasons. But I do see a recent commit adding some aria-labels, so I was wondering if accessibility is currently being worked on.

**Comment by pandeysoni** [comment]
@kyleadministrate Any update on this
